docs/quick_start.rst

```diff
@@ -145,7 +145,7 @@ Now that there are some packages in the workspace, Catkin has something to build
 .. note::
 
     Catkin utilizes an "out-of-source" and "aggregated" build pattern. This
-    means that not only will temporary or final build products never be placed
+    means that not only will temporary or final build products ever be placed
```

Comment:
I think rewording would be best. I did not want to change the meaning, only correct typos, that's why I did not commit the following:
> This means that temporary or final build products will never be placed in a package’s source directory or anywhere in the source space. Instead, all build directories are aggregated in the build space and all final build products like executables, libraries, etc., will be put in the devel space.